homework1/task1/version1.py

```diff
@@ -0,0 +1,24 @@
+# make list flatten
+# lst = [1, [2, 3], 4, [[6, 7]]]
+# to
+# lst = [1, 2, 3, 4, 6, 7]
+
+# solution from: https://www.geeksforgeeks.org/python-convert-a-nested-list-into-a-flat-list/
+# using recursion
+
+
+input_list = [1, [2, 3], 4, [[6, 7]]]
+output_list = []
+
+
+def remove_nestings(input_lst):
+    for i in input_lst:
+        if type(i) == list:
```

Comment:
'if isinstance(i, list)` works best here.
With isinstance you can also check if variable's class is a subclass of requested type

Reply:
Thank you. Code is changed